### PR TITLE
implement replyTo in pimcore mail, see #2351

### PR DESCRIPTION
**Pull request by solverat**
## ReplyTo in Pimcore Mail #2351 
Allows to define replyTo Addresses in Pimcore Mail.

## Todo:
- [x] **Translation**: Should i update the backend json files? `email_reply_to` needs to be added.
- [ ] **Update Script**:
```sql
ALTER TABLE email_log ADD COLUMN `replyTo` VARCHAR(255) DEFAULT NULL AFTER `from`;
ALTER TABLE documents_email ADD COLUMN `replyTo` VARCHAR(255) DEFAULT NULL AFTER `from`;
```

### CODE REVIEW

**Comment by fashxp** [comment]
translation in en.json is needed, other languages come from poeditor.

**Comment by solverat** [comment]
@fashxp done!

**Comment by fashxp** [comment]
Thx, we'll merge that after Pimcore 5.1. We are quite busy now an want to meet the release date.

**Comment by maff** [comment]
@solverat could you prepare an update script for build 168?

**Comment by solverat** [comment]
@maff done!

File: web/pimcore/static6/js/pimcore/settings/email/log.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PEL
  */
 
-pimcore.registerNS("pimcore.settings.email.log");
+pimcore.registerNS('pimcore.settings.email.log');
```
Comment (maff) [inline web/pimcore/static6/js/pimcore/settings/email/log.js]: Please avoid such formatting stuff (quote change throughout the file) as it makes review of the actual changes done in a PR way harder. Happy to accept formatting/style changes, but in a dedicated change. Anyways, merging this as it is :)

Comment (solverat) [reply web/pimcore/static6/js/pimcore/settings/email/log.js]: understand!

**Pull request merged**